GoogleUtilities/ISASwizzler/GULSwizzledObject.m

```diff
@@ -61,4 +62,28 @@ - (BOOL)respondsToSelector:(SEL)aSelector {
   return [gulClass instancesRespondToSelector:aSelector] || [super respondsToSelector:aSelector];
 }
 
+- (void)dealloc {
+  // We need to make sure the swizzler is deallocated after the swizzled object to do the clean up
```

Comment:
Make it explicit that this is the reason that ARC is disabled for building this file.